CGAL_ImageIO/src/CGAL_ImageIO/gis_impl.h

```diff
@@ -249,7 +249,7 @@ CGAL_INLINE_FUNCTION
 int readGisHeader( const char* name,_image* im)
 {
   char *s, *str = NULL;
-  int status;
```

Comment:
Here there is an error. `status` is used in:
  - `status = iss.str().length()`, which is fine,
  - `status = sscanf(...)` but here *there is an error*.

The return type of `sscanf` has to be signed! A correct version of the code would:
  - let `status` be of type `int`,
  - use another `size_t` variable to store the value  `iss.str().length()`.